assets/js/modules/adsense/components/dashboard/AdSensePerformanceWidget.js

```diff
@@ -71,23 +72,38 @@ class AdSensePerformanceWidget extends Component {
 
 	render() {
 		const {
-			twentyEightDays,
-			prev28Days,
+			currentRangeData,
+			prevRangeData,
 		} = this.state;
 
-		const dataBlocks = twentyEightDays.totals ? [
+		const dataBlocks = currentRangeData.totals ? [
+			{
+				className: 'googlesitekit-data-block--page-rpm',
+				title: __( 'Earnings', 'google-site-kit' ),
+				datapoint: readableLargeNumber( currentRangeData.totals[ 0 ], currentRangeData.headers[ 0 ]?.currency ),
+				change: ( ! isUndefined( prevRangeData.totals ) ) ? changeToPercent( prevRangeData.totals[ 0 ], currentRangeData.totals[ 0 ] ) : 0,
+				changeDataUnit: '%',
+			},
 			{
 				className: 'googlesitekit-data-block--page-rpm',
 				title: __( 'Page RPM', 'google-site-kit' ),
-				datapoint: readableLargeNumber( twentyEightDays.totals[ 1 ] ),
-				change: ( ! isUndefined( prev28Days.totals ) ) ? prev28Days.totals[ 1 ] : 0,
+				datapoint: readableLargeNumber( currentRangeData.totals[ 1 ], currentRangeData.headers[ 1 ]?.currency ),
+				change: ( ! isUndefined( prevRangeData.totals ) ) ? changeToPercent( prevRangeData.totals[ 1 ], currentRangeData.totals[ 1 ] ) : 0,
 				changeDataUnit: '%',
 			},
 			{
 				className: 'googlesitekit-data-block--impression',
 				title: __( 'Impressions', 'google-site-kit' ),
-				datapoint: readableLargeNumber( twentyEightDays.totals[ 2 ] ),
-				change: ! isUndefined( prev28Days.totals ) ? prev28Days.totals[ 2 ] : 0,
+				datapoint: readableLargeNumber( currentRangeData.totals[ 2 ] ),
+				change: ( ! isUndefined( prevRangeData.totals ) ) ? changeToPercent( prevRangeData.totals[ 2 ], currentRangeData.totals[ 2 ] ) : 0,
+				changeDataUnit: '%',
+			},
+			{
+				className: 'googlesitekit-data-block--impression',
+				title: __( 'Page CTR', 'google-site-kit' ),
+				/* translators: %s: percentage value. */
+				datapoint: sprintf( _x( ' %1$s%%', 'AdSense performance Page CTA percentage', 'google-site-kit' ), currentRangeData.totals[ 3 ] * 100 ),
```

Comment:
@felixarntz just wanted to call this out. The formatting of numbers followed by % signs can be different per language. For example, English is 50% while many other languages would display as 50 %.

Reply:
@ryanwelcher Thanks for pointing this out. Let's open a follow-up issue to address this throughout the plugin. We have many cases where we pass the unit independently, so this would involve a bit more work.

Reply:
@felixarntz I've opened #2200 to track this.